internal/controller/feature/features.go

```diff
@@ -46,6 +46,11 @@ const (
 	//
 	// AdditionalCertificateOutputFormats enable output additional format
 	AdditionalCertificateOutputFormats featuregate.Feature = "AdditionalCertificateOutputFormats"
+
+	// alpha: v1.8.0
+	//
+	// ServerSideApply enables the use of ServerSideApply in all API calls.
```

Comment:
> in all API calls

I guess this isn't strictly true, we only use it when updating 'status' subresource

Reply:
Hmm, no we also use it for updating some resource's object meta, secret's data etc.